operator/autoscaler_test.go

```diff
@@ -341,15 +341,36 @@ func TestScaleUpCausedByShardToNodeRatioExceeded(t *testing.T) {
 	require.Equal(t, UP, actual.ScalingDirection, actual.Description)
 }
 
+func TestScaleDownCausedByShardToNodeRatioLessThanOne(t *testing.T) {
+	eds := edsTestFixture(12)
+	esNodes := make([]ESNode, 0)
+
+	// require to scale-up because we exceeded shard-to-node ratio limits.
```

Comment:
change to 'require to scale-up index replicas because we are below one shard per node.'